packages/components/select/src/select.vue

```diff
@@ -152,11 +152,7 @@
                 aria-haspopup="listbox"
                 @focus="handleFocus"
                 @blur="handleBlur"
-                @keydown.down.stop.prevent="navigateOptions('next')"
-                @keydown.up.stop.prevent="navigateOptions('prev')"
-                @keydown.esc.stop.prevent="handleEsc"
-                @keydown.enter.stop.prevent="selectOption"
-                @keydown.delete.stop="deletePrevTag"
+                @keydown.prevent.stop="handleKeydown"
```

Comment:
`@keydown.delete.stop` does not seem to have the `prevent` modifier bound, which may change the original logic. 🤔